LLMonFHIR/FHIRInterpretation/UserStudy/SurveyWelcomeView.swift

```diff
@@ -0,0 +1,239 @@
+                    self.persistUserStudyOpenApiToken()
+                    await standard.fetchRecordsFromHealthKit()
+                    await fhirInterpretationModule.updateSchemas()
+                }
+        }
+    }
+
+
+    private var mainContent: some View {
+        VStack {
+            Spacer()
+            studyLogo
+            studyInformation
+            Spacer()
+            bottomSection
+        }
+    }
+
+    private var studyLogo: some View {
+        Image("StanfordBlockSTree")
+            .resizable()
+            .scaledToFill()
+            .frame(width: 100, height: 100)
+            .accessibilityLabel(Text("Official Stanford Logo. Block S with Tree."))
+    }
+
+    private var studyInformation: some View {
+        VStack(spacing: 24) {
+            studyTitle
+            studyDescription
+        }
+        .padding(.top, 48)
+    }
+
+    private var studyTitle: some View {
+        VStack(spacing: 8) {
+            Text(survey.title)
+                .font(.title)
+                .fontWeight(.bold)
+                .foregroundColor(.primary)
+            Text("LLM_ON_FHIR")
+                .font(.title2)
+                .foregroundColor(.secondary)
+        }
+    }
+
+    private var studyDescription: some View {
+        Text(survey.explainer)
+            .font(.body)
+            .multilineTextAlignment(.center)
+            .foregroundColor(.secondary)
+            .fixedSize(horizontal: false, vertical: true)
+            .padding(.horizontal, 32)
+    }
+
+    @ViewBuilder private var recordsStartDateView: some View {
+        if let earliestRecordDateFormatted {
+            Button {
+                isPresentingEarliestHealthRecords = true
+            } label: {
+                Text("HEALTH_RECORDS_SINCE: \(earliestRecordDateFormatted)")
+                    .font(.caption)
+                    .foregroundColor(.secondary)
+                    .fontWeight(.medium)
+                    .underline()
+            }
+            .opacity(waitingState.isWaiting ? 0 : 1)
+            .padding(.bottom, 16)
+        }
+    }
+
+    private var bottomSection: some View {
+        VStack(spacing: 16) {
+            startStudyButton
+                .padding(.horizontal, 32)
+            recordsStartDateView
+            approvalBadge
+        }
+        .padding(.bottom, 24)
+    }
+
+    private var startStudyButton: some View {
+        Group {
+            if #available(iOS 26.0, *) {
+                _startStudyButton
+                    .buttonStyle(.glassProminent)
+            } else {
+                _startStudyButton
+                    .background(Color.accent.opacity(waitingState.isWaiting ? 0.5 : 1))
+                    .cornerRadius(16)
+                    .buttonStyle(.borderedProminent)
+            }
+        }
+    }
+    
+    private var _startStudyButton: some View {
+        Button {
+            interpreter.startNewConversation()
```

Comment:
We should move the QR code scanning to this point; ensures that a user can go through the app onboarding by themselves, ideally already connect their health records, and then land at the home screen of the app to be able to wait for the study coordinator.

From there, one should be able to scan the QR code (might be in place of this button), and then be able to go to the study (maybe only then the "Start Study") button is enabled. Before that it might just be disabled. We also need to ensure that the schema is fully updated (force update) when the `startNewConversation` call is made, ideally right in that function if we don't already do that, especially when we inject a prompt from the study and something similar.